patches/third_party-blink-renderer-platform-runtime_enabled_features.json5.patch

```diff
@@ -1,24 +1,36 @@
 diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
-index 515dcf9b89e953bf339de948e82982693552ada8..810855d42333f9b7efbee7fb1e878c18fade8f2b 100644
```

Comment:
This patch could be removed, once we get https://chromium-review.googlesource.com/c/chromium/src/+/6731644